main.py

```
@@ -2780,23 +2730,13 @@ async def on_message(message: discord.Message) -> None:
-                    suffix_string += f"\n{get_emoji('insane')} This message appears on ***0.000001%*** of catches!!!"
-                elif randnum % 10000000 == 0:
-                    suffix_string += f"\n{get_emoji('rainbow_sparkles')} This message appears on **0.00001%** of catches!!!"
-                elif randnum % 1000000 == 0:
-                    suffix_string += "\n💫 This message appears on **0.0001%** of catches!"
-                elif randnum % 100000 == 0:
-                    suffix_string += "\n🌟 This message appears on *0.001%* of catches!"
-                elif randnum % 10000 == 0:
-                    suffix_string += "\n✨ This message appears on *0.01%* of catches."
-                elif randnum % 1000 == 0:
-                    suffix_string += "\n⭐ This message appears on 0.1% of catches."
+                sparkle_roll = random.random()
```

Ruff (0.16.1)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)